capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/camera/photo/PhotoEdit.java

```diff
@@ -73,6 +85,39 @@ private void removeCompressionModifier() {
         }
     }
 
+    public PhotoEdit crop(Activity context, Rect aRect) {
+
+        byte[] originalBytes = mPhoto.getData();
+
+        try {
+            Bitmap originalBitmap = BitmapFactory.decodeByteArray(originalBytes,
+                    0, mPhoto.getData().length);
+
+            DisplayMetrics metrics = new DisplayMetrics();
+            context.getWindowManager().getDefaultDisplay().getMetrics(metrics);
```

Comment:
[Suggestion] The display size does not account for the navigation bar. Can you use the `gc_camera_preview_container` to get the size that accounts for the navigation bar?